src/Uno.UI.Tests/Windows_UI_XAML_Controls/GridTests/Given_Grid.cs

```diff
@@ -23,142 +15,174 @@ public class Given_Grid : Context
 		[TestMethod]
 		public void When_Empty_And_MeasuredEmpty()
 		{
-			var SUT = new Grid() { Name = "test" };
+			var SUT = new Grid { Name = "test" };
 
-			SUT.Measure(default(Windows.Foundation.Size));
+			SUT.Measure(default);
 			var size = SUT.DesiredSize;
-			SUT.Arrange(default(Windows.Foundation.Rect));
+			SUT.Arrange(default);
 
-			Assert.AreEqual(default(Windows.Foundation.Size), size);
+			Assert.AreEqual(default, size);
 			Assert.IsTrue(SUT.GetChildren().None());
 		}
 
 		[TestMethod]
 		public void When_Empty_And_Measured_Non_Empty()
 		{
-			var SUT = new Grid() { Name = "test" };
+			var SUT = new Grid { Name = "test" };
 
-			SUT.Measure(new Windows.Foundation.Size(10, 10));
+			SUT.Measure(new Size(10, 10));
 			var size = SUT.DesiredSize;
-			SUT.Arrange(default(Windows.Foundation.Rect));
+			SUT.Arrange(default);
 
-			Assert.AreEqual(size, default(Windows.Foundation.Size));
+			Assert.AreEqual(size, default(Size));
 			Assert.IsTrue(SUT.GetChildren().None());
 		}
 
 		[TestMethod]
 		public void When_Grid_Has_One_Element()
 		{
-			var SUT = new Grid() { Name = "test" };
+			var SUT = new Grid { Name = "test" };
 
-			SUT.AddChild(new View { Name = "Child01", RequestedDesiredSize = new Windows.Foundation.Size(10, 10) });
+			SUT.AddChild(new View { Name = "Child01", RequestedDesiredSize = new Size(10, 10) });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define a constant instead of using this literal 'Child01' 27 times.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826507)